Documentation/tpv12.rst

```diff
@@ -1,115 +1,89 @@
 SCEC TPV12
 ==========
 
-TPV12 and 13 are recommended by SCEC for elastic/plastic wave
-propagation code validation. TPV 12 describes spontaneous rupture on a
-**60-degree dipping normal fault** in a homogeneous half-space. Material
-properties are linear **elastic**. Initial stress conditions are
-dependent on depth. Strongly super-shear rupture conditions.
+TPV12 and 13 are verification benchmarks for  a 60 degree dipping normal fault embedded in a homogeneous halfspace.
+The rheology is linear elastic in TPV12 and non-associative Drucker-Prager visco-plastic in TPV13.
+Initial stress conditions are
+dependent on depth. Strongly super-shear rupture conditions are assumed.
 
 .. figure:: LatexFigures/tpv12_13.png
-   :alt: Diagram of geometry of TPV12/13.
+   :alt: Geometry in SCEC benchmarks TPV12/13
    :width: 9.00000cm
 
-   Diagram of the geometry of TPV12/13. 60-degree dipping normal fault.
+   Figure 1: Geometry in SCEC benchmarks TPV12/13, including a 60-degree dipping normal fault.
 
 Geometry
 ~~~~~~~~
 
-The model volume is a half-space. The fault is a 60-degree dipping,
-planar, normal fault. The fault reaches the Earth’s surface. Rupture is
-allowed within a rectangular area measuring 30000 m along-strike and
-15000 m down-dip.
-
-Note that 15000 m down-dip corresponds to a depth of 12990.38 m. A node
-which lies exactly on the border of the 30000 m :math:`\times` 15000 m
-rectangle is considered to be inside the rectangle, and so should be
-permitted to rupture.
-
-The portions of the fault below, to the left of, and to the right of the
-30000 m :math:`\times` 15000 m rectangle are a strength barrier, within
-which the fault is not allowed to rupture.
-
-The nucleation zone is a square measuring 3000 m × 3000 m. The center of
-the square is located 12000 m down-dip (at a depth of 10392.30 m), and
-is centered along-strike.
-
-The geometry is generated with `Gmsh <https://gmsh.info/>`_. All the files that are needed for
+A half-space domain is assumed. The fault is a 60-degree dipping, 30 km :math:`\times` 15 km
+planar, normal fault. It reaches the Earth’s surface, and extends up to 12.99038 km depth.
+A square nucleation area of 3 :math:`\times` 3 km is assumed.
+In fault coordinates, it is centered at (0, -12) km, that is at a depth of 10.3923 km.
+The CAD model (see :ref:`Figure 2 <TP12_figure_2>`) and mesh are generated with `Gmsh <https://gmsh.info/>`_. All the files that are needed for
 the simulation are provided at https://github.com/SeisSol/Examples/tree/master/tpv12_13.
```

Comment:
It is just a suggestion. Could we write something like 
```
All the files that are needed for
the simulation are provided `here <https://github.com/SeisSol/Examples/tree/master/tpv12_13>`_.
```

Then it will appear as
```
All the files that are needed for the simulation are provided here.
```
where `here` is going to be highlighted as a clickable URL reference.

Reply:
ok I did that